### ISSUE TITLE
Enhancement Request - Battery Status Icon

### ISSUE THREAD

**Issue by dVIAN13** [JohnCoates/Aerial#980]
@glouel, a HUGE thank you for your continued development and the MANY improvements you continue to bring to this project! 

The new weather functionality is awesome, and the addition of the icons makes it look beautiful.

I had an enhancement request, if possible.. :) Would it be possible to bring an icon to the battery status? I am thinking in such a way as the below icons (example/screenshot taken from the macOS Battery Indicator app from Sindre Sorhus) 

Plugged-in, Charging
<img width="39" alt="Screen Shot 2020-05-01 at 8 01 06 AM" src="https://user-images.githubusercontent.com/22992333/80815187-04289800-8b82-11ea-92b2-ca59d5814c66.png">
Plugged-in, Fully Charged
<img width="33" alt="Screen Shot 2020-04-19 at 10 37 59 AM" src="https://user-images.githubusercontent.com/22992333/80815190-0559c500-8b82-11ea-8a94-7b242f2107b3.png">
On Battery
<img width="37" alt="Screen Shot 2020-04-19 at 10 38 07 AM" src="https://user-images.githubusercontent.com/22992333/80815191-0559c500-8b82-11ea-8bd6-7a350cbad4bd.png">

**Comment by glouel** [comment]
Hey @dVIAN13 !

I had put the icon aside as I had issues adding icons previously. I fixed that with the weather stuff in beta so that should be coming very soon, will let you know ;)

**Comment by glouel** [comment]
@dVIAN13 please try beta9

![Capture d’écran 2020-05-01 à 19 34 14](https://user-images.githubusercontent.com/37544189/80828868-500d2880-8be6-11ea-953e-366989eda64c.jpg)

https://github.com/JohnCoates/Aerial/releases/download/v1.8.3beta9/Aerial.saver.zip

**Comment by dVIAN13** [comment]
@glouel 🥳🥳 you are amazing! looks fantastic!

**Comment by glouel** [comment]
@dVIAN13 Thanks again for what you sent me, you've been quite generous over time, it's truly appreciated!

I did change a couple of things in the new beta, I made the indicator slightly larger as it wasn't really readable on non retina screens. I could maybe have different size available in options, although it might be too much. 

I also changed the bolt icon for a filled one, it looks better in my opinion (and on non retina the old one was truly horrible). 

<img width="208" alt="Capture d’écran 2020-05-02 à 17 59 33" src="https://user-images.githubusercontent.com/37544189/80869628-d9cdfc00-8ca1-11ea-8ab5-9ef9b84d0137.png">

(I still need to fix the margins on images, as having both Weather and battery on looks a bit weird ;)

https://github.com/JohnCoates/Aerial/releases/download/v1.8.3beta10/Aerial.saver.zip

**Comment by dVIAN13** [comment]
@glouel, As always, you deserve it! As to your point about margins, I observed the same spacing and decided to just move the weather to a different corner. :) Thank you for changing the charging bolt icon. I agree it is much easier to see now.

**Comment by glouel** [comment]
@dVIAN13 I just released 1.9.0 with the latest changes. Regarding the icon alignments, it was a bit of a mess so I ended up rewriting everything. It should now be properly aligned, both in terms of marins of an icon to an icon or an icon to text when on top of each other. 

And also, they align fine on the left side too, which wasn't the case previously. Every corner should be ok now :) I also ended up removing the text option for the battery indicator as it was more of a stopgap than anything until I could finally add icons. 

<img width="357" alt="Capture d’écran 2020-05-25 à 14 45 15" src="https://user-images.githubusercontent.com/37544189/82816034-b2caaa80-9e9a-11ea-9e06-53656c6e92f0.png">

https://github.com/JohnCoates/Aerial/releases/download/v1.9.0/Aerial.saver.zip
